primary.tf

```diff
@@ -39,14 +39,16 @@ resource "google_compute_instance" "primary" {
     cluster-api-endpoint = "${var.prefix}-primary-0-${random_string.postfix.result}:6443"
     primary-pki-url      = "http://${var.prefix}-primary-0-${random_string.postfix.result}:${local.assistant_port}/api/v1/pki-download?token=${random_string.setup_token.result}"
     health-url           = "http://${var.prefix}-primary-0-${random_string.postfix.result}:${local.assistant_port}/healthz"
+    assistant-host       = "http://${var.prefix}-primary-0-${random_string.postfix.result}:${local.assistant_port}"
+    assistant-token      = "${random_string.setup_token.result}"
     custom-ca-cert-url   = "${var.ca_bundle_url}"
     ptfe-role            = "${count.index == 0 ? "main" : "primary"}"
     role-id              = "${count.index}"
     ptfe-install-url     = "${var.ptfe_install_url}"
     jq-url               = "${var.jq_url}"
     b64-license          = "${base64encode(file("${var.license_file}"))}"
     airgap-package-url   = "${var.airgap_package_url}"
-    airgap-installer-url = "${var.airgap_installer_url}"
+    airgap-installer-url = "${var.airgap_package_url == "none" ? "none" : count.index == 0 ? var.airgap_installer_url : local.internal_airgap_url}"
```

Comment:
Using `"none"` as a default is odd but this is consistent.

Reply:
This is due to a super fun quirk of RHEL on GCP where it ignores null metadata, which results in files missing entirely!